client/modules/User/components/CookieConsent.jsx

```diff
@@ -175,10 +175,13 @@ function CookieConsent({ hide }) {
                   kind={Button.kinds.secondary}
                   onClick={acceptAllCookies}
                 >
-                  {t('Cookies.AllowAll')}
+                  <>{t('Cookies.AllowAll')}</>
```

Comment:
In my opinion it would be better to to handle this in the `Button` by allowing `string` as a valid `children` type.  This can be done with `children: PropTypes.node`.